Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -178,6 +178,18 @@ def keg_contain?(string, keg, ignores)
           offset, match = str.split(" ", 2)
           next if linked_libraries.include? match # Don't bother reporting a string if it was found by otool
 
+          # Do not report matches to directories that do not exist.
```

Comment:
Could this just check for paths that don't exist? Seems like it might be more specific and therefore more likely to find `cellar :any` relocations.

Reply:
Sure